<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>iPainter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="label_color_foreground">
        <property name="text">
         <string>前景</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="btn_tool_rectangle">
        <property name="text">
         <string>矩形</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="btn_tool_circle">
        <property name="text">
         <string>椭圆</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="slider_stroke_width_select">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="btn_tool_filler">
        <property name="text">
         <string>填充</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="btn_tool_line">
        <property name="text">
         <string>直线</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="btn_tool_pencil">
        <property name="text">
         <string>铅笔</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="btn_tool_eraser">
        <property name="text">
         <string>橡皮</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="btn_tool_airbrush">
        <property name="text">
         <string>喷笔</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_color_background">
        <property name="text">
         <string>背景</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="btn_tool_text">
        <property name="text">
         <string>文字</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="7">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="painter_area">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>618</width>
           <height>541</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="paper">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="label_color_foreground_show">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QToolButton" name="label_color_background_show">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action__file_new"/>
    <addaction name="action__file_open"/>
    <addaction name="action__file_save"/>
    <addaction name="separator"/>
    <addaction name="action__file_quit"/>
   </widget>
   <widget class="QMenu" name="menu_filter">
    <property name="title">
     <string>效果</string>
    </property>
    <addaction name="action__filter_blur"/>
    <addaction name="action__filter_sharpen"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>动作</string>
    </property>
    <addaction name="action__action_undo"/>
    <addaction name="action__action_redo"/>
    <addaction name="separator"/>
    <addaction name="action__action_change_image_shape"/>
    <addaction name="action__action_image_rescaling"/>
    <addaction name="separator"/>
    <addaction name="action__action_rotate"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
   <addaction name="menu_filter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action__file_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action__file_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action__file_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action__filter_blur">
   <property name="text">
    <string>模糊</string>
   </property>
  </action>
  <action name="action__filter_sharpen">
   <property name="text">
    <string>锐化</string>
   </property>
  </action>
  <action name="action__file_new">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action__action_undo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action__action_redo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="action__action_change_image_shape">
   <property name="text">
    <string>画布缩放</string>
   </property>
  </action>
  <action name="action__action_image_rescaling">
   <property name="text">
    <string>图像缩放</string>
   </property>
  </action>
  <action name="action__action_rotate">
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
